src/main/kotlin/net/lumalyte/lg/infrastructure/services/NexoEmojiService.kt

```
@@ -109,12 +113,103 @@ class NexoEmojiService(
+
+    /**
+     * Looks up a Nexo glyph object by name via FontManager reflection.
+     *
+     * @param name glyph id without colons (e.g. `catsmileysmile`).
+     * @return the glyph object, or null if Nexo/FontManager is unavailable or the lookup fails.
+     */
+    private fun resolveGlyphByName(name: String): Any? {
+        val fontManager = getFontManager() ?: return null
+        return try {
+            fontManager.javaClass.getMethod("glyphFromName", String::class.java).invoke(fontManager, name)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)